.typos.toml

```diff
@@ -17,6 +18,8 @@ MDEyOk9yZ2FuaXphdGlvbjg4Njc2OTU1 = "MDEyOk9yZ2FuaXphdGlvbjg4Njc2OTU1"
 
 [default.extend-words]
 # Don't correct the surname "Teh"
+preprocesser = "preprocesser"
+Preprocesser = "Preprocesser"
```

Comment:
shouldn't we just fix the actual spelling mistake instead of adding an exception for a 'wrong' spelling?

Reply:
Ignore this, though, if it's going to take time to fix this.

Reply:
I think you can just ignore it. #TODO comments are easily forgotten.